src/main.py

```diff
@@ -159,6 +159,7 @@ async def main() -> None:
     offset = os.environ["OFFSET"]
     kafka_schema_registry_url = os.environ["KAFKA_SCHEMA_REGISTRY_URL"]
     latiss_knative_serving_url = os.environ["LATISS_KNATIVE_SERVING_URL"]
```

Comment:
Spotted my typo in the commit message (LSSTComCam not LSSTCamCam); will fix that.